java/ql/src/Security/CWE/CWE-200/TempDirLocalInformationDisclosureFromSystemProperty.ql

```diff
@@ -0,0 +1,82 @@
+/**
+ * @name Temporary Directory Local information disclosure
+ * @description Detect local information disclosure via the java temporary directory
```

Comment:
If we could also give this a description that expands on the title using our "Syntax X causes behavior Y" pattern, that would be great.

Reply:
Sorry, need a bit more guidance here.

Reply:
Sorry - it should probably be much the same as the description for the other query.